bundle/Sail-Operator-ReadMe.md

```diff
@@ -0,0 +1,54 @@
+# About the Sail Operator
+
+The present implementation is based on the open source Istio project. The implementation pulls its code from the upstream Istio master with no changes to the codebase.
+
+# Prerequisites
+
+You have deployed a cluster on OpenShift Container Platform 4.13 or later.
+
+# Installing the Sail Operator
+
+1. Create the project where Istio is going to be deployed:  
+
+``` 
+$ oc new-project istio-system
+```
+
+2. And give it privileges (workaround while anyuid upstream PR is not merged)  
```

Comment:
remove what's in the parenthesis. When it gets merged upstream we can just remove this section from this doc

Reply:
done